frontend/packages/ux-editor/src/components/config/componentSpecificContent/AttachmentList/AttachmentListUtils.ts

```diff
@@ -0,0 +1,114 @@
+import { ArrayUtils } from '@studio/pure-functions';
+
+export const reservedDataTypes = {
+  includeAll: 'include-all',
+  currentTask: 'current-task',
+  refDataAsPdf: 'ref-data-as-pdf',
+};
+
+export type AttachmentsFormat = {
+  attachmentsCurrentTasks: string[];
+  attachmentsAllTasks: string[];
+};
+
+export type InternalDataTypesFormat = {
+  currentTask: boolean;
+  includePdf: boolean;
+  selectedDataTypes: string[];
+};
+
+export type ConvertInternalToExternalFormat = {
+  availableAttachments: AttachmentsFormat;
+  dataTypeIds: InternalDataTypesFormat;
+};
+
+export const convertInternalToExternalFormat = ({
+  availableAttachments,
+  dataTypeIds,
+}: ConvertInternalToExternalFormat): string[] => {
```

Comment:
Som hovedregel er det fint å gruppere ting i objekter, men akkurat dette kan godt være to separate parametre. De har jo forholdsvis uavhengige av hverandre. Det gjelder også `convertExternalToInternalFormat`.